stylesheet/main.css

```diff
@@ -15,12 +17,25 @@ a {
 
 header nav a {
     color: #5E5369;
+    display: block;
+    padding-top: 1em;
+    padding-bottom: 0.5em;
+    margin-left: 1em;
+    margin-right: 1em;
+    border-top: 4px solid transparent;
+}
+header nav a:hover {
+    border-top-color: red;
 }
 
 header ul {
     list-style-type: none;
     margin: 0;
     padding: 0;
+    float: right;
+}
+header ul li {
+    float: left;
 }

```

Comment:
These rules have the selectors that apply to the list and the list items in the header, that were to be floated